timescaledb/overview/index.md

```diff
@@ -1,51 +1,86 @@
-# TimescaleDB overview
+# TimescaleDB Overview
 
-TimescaleDB is an open-source relational database for time-series data.  It
-speaks "full SQL" and is correspondingly easy to use like a traditional relational
+TimescaleDB is an open-source relational database for time-series data.  It 
+speaks "full SQL" and is correspondingly easy to use like a traditional relational 
 database, yet scales in ways previously reserved for NoSQL databases.
 
-Compared to the trade-offs demanded by these two alternatives
-(relational vs. NoSQL), TimescaleDB offers the best of both
-worlds **for time-series data:**
+## Supercharged PostgreSQL
 
-## Easy to use
+Rely on the same PostgreSQL you know and love, with full SQL, 
+rock‑solid reliability, and the largest ecosystem of developer and management tools.
 
-- **Full SQL interface** for all SQL natively supported by
-PostgreSQL (including secondary indexes, non-time based aggregates,
-sub-queries, JOINs, window functions).
-- **Connects** to any client or tool that speaks PostgreSQL, no changes needed.
-- **Time-oriented** features, API functions, and optimizations.
-- Robust support for **Data retention policies**.
+* **Full SQL**, [powerful analytics][hyperfunctions], no restrictions
+* Leverage your **favorite PostgreSQL extensions**
+* **Your entire toolset works**: ORMs, connectors, JDBC, applications
+* **Connect Prometheus** for all your observability metrics
+* Visualize data using **[the dashboards you love][grafana]**
 
+## Accelerated performance
 
-## Scalable
+Achieve 10-100x faster queries than PostgreSQL, InfluxDB, and MongoDB. Leverage query parallelization, continuous aggregates, and other performance optimizations.
```

Comment:
```suggestion
Achieve 10-100x faster queries than PostgreSQL, InfluxDB, and MongoDB. 
Leverage query parallelization, continuous aggregates, and other 
performance optimizations.
```

Reply:
I won't reflow the others, because it's too hard to do in GH, but there are quite a few long lines 👇